regulations/static/regulations/css/less/module/paragraph-markers.less

```diff
@@ -59,5 +60,16 @@ ol {
 }
 
 .level-2 > li > p {
-    margin-top: 0;
+    margin-top: 1em;
+}
+
+// move key term headings up to center with paragraph markers
+.paragraph-marker + .key-term {
+  float: left;
+  margin-top: -5px;
```

Comment:
Ah, the height _is_ defined: https://github.com/18F/regulations-site/blob/master/regulations/static/regulations/css/less/typography.less#L450

```less
.level-1 .key-term {
    font-size: 1.375em; /* 24px / 16px = 1.5em */
    line-height: 1.5; /* 30px / 24px = 1.25 */
}
```

Since level-1 is the outlier in having a larger font size, should this rule focus on `level-1`?